Classes/ios/Scanners/MTBBarcodeScanner.m

```diff
@@ -154,12 +146,13 @@ - (instancetype)initWithPreviewView:(UIView *)previewView {
         _previewView = previewView;
         _metaDataObjectTypes = [self defaultMetaDataObjectTypes];
         _allowTapToFocus = YES;
+        [self setupSessionQueue];
```

Comment:
instead of adding this here was well just do
```
[self initWithMetadataObjectTypes:[self defaultMetaDataObjectTypes] previewView: previewView];
```
?